bagelbids/tests/test_cli_pheno.py

```diff
@@ -274,3 +273,27 @@ def test_cli_age_is_processed(runner, test_data, tmp_path):
 
     assert 20.5 == pheno["hasSamples"][0]["age"]
     assert pytest.approx(25.66, 0.01) == pheno["hasSamples"][1]["age"]
+
+
+def test_output_includes_context(runner, test_data, tmp_path):
+    runner.invoke(
+        bagel,
+        [
+            "--pheno",
+            test_data / "example2.tsv",
+            "--dictionary",
+            test_data / "example2.json",
+            "--output",
+            tmp_path,
+            "--name",
+            "my_dataset_name",
+        ],
+    )
+
+    with open(tmp_path / "pheno.jsonld", "r") as f:
+        pheno = json.load(f)
+
+    assert pheno.get("@context") is not None
+    assert all(
+        [sub.get("identifier") is not None for sub in pheno["hasSamples"]]
```

Comment:
Worth using this library https://github.com/digitalbazaar/pyld to add assertions that our namespace shorthands expand properly?

Reply:
> expand properly

I'm not sure what you mean by that. If you'd like to test the `generate_context` function to make sure that everything in `models` has a corresponding context entry, then I think that could be a good thing to test. But I'd probably group that with the refactor.

Reply:
Okay, I think that would be sufficient. Will do

Reply:
Ah I see. Yeah, let's think about it some more - it's a good idea to test that everything that needs to be in the context makes it there, and also that we don't add more things than necessary. When we handle our part, we can rely on json-ld to do its part correctly.